Automattic/jetpack
# VideoPress: WebVTT file content generation from video description chapters

Closes #26172 (moved from VideoPress package to Jetpack plugin)
Closes #26189

#### Changes proposed in this Pull Request:

* Moves the files from #26181 to Jetpack plugin
* Adds helper functions to generate the contents of a webVTT file of video chapters

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run `jetpack test plugins/jetpack js`
* Check that `extensions/blocks/videopress/video-chapters/utils/test/extract-video-chapters.ts` passes
* Check that `extensions/blocks/videopress/video-chapters/utils/test/generate-chapters-file-content.ts` passes

Note: Some unrelated tests are not passing.
